microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -146,42 +172,42 @@ The response value from `navigator.userAgent` method uses the following format.
+Sometimes, a website doesn't recognize the new Microsoft Edge user agent. As a result, a set of the features of the website may not work correctly. When Microsoft is notified about the types of issues, Microsoft contacts you \(a website owner\) and informs you about the updated user agent.
+
+You may need more time to update and test the user agent detection logic for your website to address the issues reported by Microsoft. To maximize compatibility for your users, the Microsoft Edge Beta and Stable channels use a list of user agent overrides. Use the user agent overrides while you update your website. The list of user agent overrides is provided by Microsoft.  
+
+The overrides specify new user agent values that Microsoft Edge sends instead of the default user agent for specific websites. To display the list of user agent overrides that are currently applied, complete the following actions.  
 
-You may need more time to update and test the user agent detection logic for your website to address the issues reported by Microsoft.  To maximize compatibility for your users, the Microsoft Edge Beta and Stable channels use a list of user agent overrides.  Use the user agent overrides while you update your website.  The list of user agent overrides provided by Microsoft.  
+1. Open the Microsoft Edge Beta or Stable channel.  
+1. Navigate to `edge://compat/useragent`.  
 
-The overrides specify new user agent values that Microsoft Edge sends instead of the default user agent for specific websites.  To display the list of user agent overrides that are currently applied, complete the following actions.  
+The Microsoft Edge Canary and Dev channels don't currently receive user agent overrides. The Microsoft Edge Canary and Dev channels provide environments that use the default Microsoft Edge user agent. Use the Microsoft Edge Canary and Dev channels to reproduce issues on your website caused by the default Microsoft Edge user agent. To turn off user agent overrides in the Microsoft Edge Beta or Stable channels, complete the following actions.  
 
-1.  Open the Microsoft Edge Beta or Stable channel.  
-1.  Navigate to `edge://compat/useragent`.  
-    
-The Microsoft Edge Canary and Dev channels don't currently receive user agent overrides.  The Microsoft Edge Canary and Dev channels provide environments that use the default Microsoft Edge user agent.  Use the Microsoft Edge Canary and Dev channels to easily reproduce issues on your website caused by the default Microsoft Edge user agent.  To turn off user agent overrides in the Microsoft Edge Beta or Stable channels, complete the following actions.  
+1. Open a command prompt. For example, enter **CMD** in the Windows search text box and select the **Command Prompt** app.
+1. Copy and paste the following code snippet.  

```

Comment:
One more typo from before your edits: I think we just want people to copy the snippet below. They paste it in the next numbered step

Reply:
fixed